src/elfpy/markets.py

```diff
@@ -115,80 +203,207 @@ def check_fees(
                 + state_string
             )
 
-    def update_market(self, market_deltas):
+    def tick(self, delta_time):
+        """Increments the time member variable"""
+        self.time += delta_time
+
+    def update_spot_price(self):
+        """Update the spot price"""
+        self.spot_price = self.pricing_model.calc_spot_price_from_reserves(
+            share_reserves=self.share_reserves,
+            bond_reserves=self.bond_reserves,
+            init_share_price=self.init_share_price,
+            share_price=self.share_price,
+            time_remaining=time_utils.stretch_time(self.token_duration, self.time_stretch_constant),
+        )
+
+    def __open_short(self, trade_details):
```

Comment:
Why are you using the mangled lead instead of a single underscore?
https://docs.python.org/3/tutorial/classes.html#private-variables

Seems like this would only be relevant if open_short also existed as other class methods?

Reply:
I was just trying to make them private, but it seems like I read some bad information. I'll change these.